config/jobs/kubernetes-sigs/cluster-api-boostrap-provider-kubeadm/cluster-api-bootstrap-provider-kubeadm-presubmits.yaml

```diff
@@ -0,0 +1,16 @@
+# sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm presubmits
+presubmits:
+  kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm:
+  - name: pull-cluster-api-bootstrap-provider-kubeadm-verify
+    path_alias: "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm"
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190626-f0c7a9f-master
+        command:
+        - "./hack/verify-all.sh"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-bootstrap-provider-kubeadm
+      testgrid-tab-name: pr-verify
+      testgrid-num-columns-recent: "20"
```

Comment:
20 is now the default for presubmits (because apparently it's the minimum and an illegal default was absurd), so this line can be omitted.

Reply:
ok, i saw less than 20 erroring out for presubmits.
i guess https://github.com/kubernetes/test-infra/pull/13225/files needs an update too.

Reply:
It errored out until I ran into the same thing and fixed it in #13215 because it was silly. It should be fine now.

Reply:
updated.